src/app/analytics/analytics.factory.js

```diff
@@ -568,8 +568,7 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, envService
             donation_frequency: frequency,
             payment_type: paymentType,
             purchase_number: purchaseData.rawData['purchase-number'],
-            campaign_code: cartItem.config.CAMPAIGN_CODE !== '' ? cartItem.config.CAMPAIGN_CODE : undefined,
-            designation: 'designation'
+            campaign_code: cartItem.config.CAMPAIGN_CODE !== '' ? cartItem.config.CAMPAIGN_CODE : undefined
```

Comment:
Is designation number captured elsewhere already?

Reply:
Yeah, it's under `item_id` https://github.com/CruGlobal/give-web/blob/55e70a9b08bbfe3341a0c1f3e94751587f56989a/src/app/analytics/analytics.factory.js#L49